peer.go

```diff
@@ -996,6 +996,12 @@ out:
 				idleTimer.Reset(idleTimeout)
 				continue
 
+			// If the NodeAnnouncement has an invalid alias, then
+			// we'll log that error above and continue so we can
+			// continue to read messges from the peer.
+			case lnwire.ErrInvalidNodeAlias:
```

Comment:
should `idleTimer` be reset, as in the other cases?

Reply:
Fixed!